File: wcpelib-nukkit/src/main/java/top/wcpe/wcpelib/nukkit/server/ServerInfo.java
package top.wcpe.wcpelib.nukkit.server;

import lombok.Data;

/**
 * 服务器信息
 *
 * @author: WCPE 1837019522@qq.com
 * @create: 2021-10-22 19:52
 */
@Data
public class ServerInfo {
    private String serverName;
    private String host;
    private int port;

    public ServerInfo(String serverName, String host, int port) {
        this.serverName = serverName;
        this.host = host;
        this.port = port;
    }
}

File: wcpelib-nukkit/src/main/kotlin/top/wcpe/wcpelib/nukkit/server/ServerInfo.kt
package top.wcpe.wcpelib.nukkit.server

/**
 * 由 WCPE 在 2022/6/3 1:53 创建
 *
 * Created by WCPE on 2022/6/3 1:53
 *
 * GitHub  : https://github.com/wcpe
 * QQ      : 1837019522
 * @author : WCPE
 * @since  : v1.0.12-alpha-dev-5
 */
data class ServerInfo(
    val serverName: String,
    val host: String,
    val port: Int,
    val viewName: String
)